### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by artyomwhite**
https://screenpal.com/watch/c0QtI3V5joQ

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strict": false,
-    "suppressImplicitAnyIndexErrors": true,
```
Comment (andriy-shymkiv) [inline tsconfig.json:23]: but why did you turned off this rule? do you have some ts errors?

**Review by andriy-shymkiv — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (artyomwhite) [reply tsconfig.json:23]: Screenshot_1

Comment (artyomwhite) [reply tsconfig.json:23]: mentor advised to remove this rule, but if you have a solution to this problem I would love to hear from you!
Screenshot_2

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**